.github/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 <h3>New features since last release</h3>
 
+* Added a new bit-flip mixer to the `qml.qaoa` module.
+  [(#774)](https://github.com/PennyLaneAI/pennylane/pull/774)
+
```

Comment:
More out of curiosity - what makes this a new feature vs an improvement?

Reply:
Yeah, I think it could be either. However, since this is technically an addition to the pre-existing collection of mixers, I can see how it would be an improvement 👍